cogs/custom.py

```diff
@@ -38,14 +42,73 @@ async def vote(self, ctx: Context, title: str, *options: str) -> None:
             raise BadArgument("Please provide at least 2 options.")
         if len(options) > 20:
             raise BadArgument("I can only handle 20 options!")
-
         codepoint_start = 127462  # represents "regional_indicator_a" unicode value
         options = {chr(i): f"{chr(i)} - {v}" for i, v in enumerate(options, start=codepoint_start)}
         embed = Embed(title=title, description="\n".join(options.values()))
         message = await ctx.send(embed=embed)
         for reaction in options:
             await message.add_reaction(reaction)
 
+    @command()
+    async def members(self, ctx: Context) -> None:
+        """Returns the number of members in a server."""
+        await ctx.send(embed=Embed(title="Member count", description=ctx.guild.member_count, color=Color.dark_purple()))
+
+    @command()
+    async def status(self, ctx: Context, member: Member = None) -> None:
+        """Returns the status of a member."""
+        if member is None:
+            member = ctx.author
+
+        if member.id == constants.owner_id:
+            embed = status_embed(member, description="Not telling")
+        else:
+            embed = status_embed(member)
+
+        await ctx.send(embed=embed)
```

Comment:
I'm not sure why is this useful, discord already shows you the status. I don't see a need for a command.
```suggestion
```

Reply:
This is for the above members function